packages/utilities/src/dateTimeFormat/absolute.ts

```diff
@@ -40,6 +43,7 @@ export function format(
     style: Intl.DateTimeFormatOptions['timeStyle'] | 'tooltip';
     timeStyle: Intl.DateTimeFormatOptions['timeStyle'];
     dateStyle: Intl.DateTimeFormatOptions['dateStyle'];
+    timeZone?: Intl.DateTimeFormatOptions['timeZone'];
   }>
```

Comment:
`options` is already wrapped in `Partial<>`, so the `?` after `timeZone` is redundant.